app/components/date_choice_facet_component.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= render(@layout.new(facet_field: @facet_field)) do |component| %>
+  <% component.with(:label) do %>
+    <%= @facet_field.label %>
+  <% end %>
+  <% component.with(:body) do %>
+    <ul class="facet-values list-unstyled">
+      <%= render_facet_limit_list @facet_field.paginator, @facet_field.key %>
+    </ul>
```

Comment:
I think ideally you'd be able to do:

```suggestion
    <%= render Blacklight::FacetFieldListComponent.new(@facet_field, layout: false)
```

I don't know if the `more >` link that's in there for backwards compat. presents a problem (and maybe the backwards compat needs to be explicit instead of keying off the layout?)